package.json

```diff
@@ -117,6 +117,7 @@
     "react-native-reanimated": "^2.3.1",
     "react-native-safe-area-context": "^3.1.8",
     "react-native-screens": "^2.12.0",
+    "react-native-shared-group-preferences": "^1.1.23",
```

Comment:
Could you add `"@types/react-native-shared-group-preferences": "^1.1.1",` to the dev dependencies as well?

Reply:
It's added as a dependency so that isn't necessary

Reply:
This is the types package. It is needed for Typescript support for the library.

Reply:
Oh I see sorry misread